packages/cardpay-sdk/sdk/scheduled-payment-module.ts

```diff
@@ -239,4 +240,96 @@ export default class ScheduledPaymentModule {
 
     return requiredGas;
   }
+
+  async cancelScheduledPayment(txnHash: string): Promise<SuccessfulTransactionReceipt>;
+  async cancelScheduledPayment(
+    safeAddress: string,
+    moduleAddress: string,
+    spHash: string,
+    gasTokenAddress: string,
+    txnOptions?: TransactionOptions,
+    contractOptions?: ContractOptions
+  ): Promise<void>;
+  async cancelScheduledPayment(
+    safeAddressOrTxnHash: string,
+    moduleAddress?: string,
+    spHash?: string,
+    gasTokenAddress?: string,
+    txnOptions?: TransactionOptions,
+    contractOptions?: ContractOptions
+  ): Promise<SuccessfulTransactionReceipt | void> {
+    if (isTransactionHash(safeAddressOrTxnHash)) {
+      let txnHash = safeAddressOrTxnHash;
+      return await waitUntilTransactionMined(this.layer2Web3, txnHash);
```

Comment:
i just noticed that the web3 instance variable is called `layer2Web3`. i think that is not totally accurate. We used layer1Web3 and layer2Web3 in the previous card pay work because it was really important to be clear aboutwhat layer the web3 instance was. for all the scheduled payment work, there is no special need to have a separate web3 instance for each layer. I think we can just call this class property `this.web3`. (same for all the other schedule payment related SDK classes)

Reply:
updated